docker/Docker_amazonlinux2

```diff
@@ -0,0 +1,16 @@
+FROM amazonlinux:2
+
+ARG API_KEY
+ARG BASE_DN
+ARG BIND_USER
+ARG BIND_PASSWORD
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN yum -y update && yum -y install epel-release python-pip openssh-server sudo
+#RUN yum --enablerepo=epel install -y python-pip
+RUN python -m pip install urllib3
+
+COPY linux/amzn/2.0/foxpass_setup.py /
+
+ENTRYPOINT python foxpass_setup.py --base-dn {BASE_DN} --bind-user {BIND_USER} --bind-pw {BIND_PW} --api-key {API_KEY} --enable-ldap-sudoers --debug
```

Comment:
Could we also add something that basically hints that the setup is working fine. This runs the script, but we still wouldn't know if the integration works fine. We can add a script after running the script that ensures the setup works.

Reply:
Resolved as discussed.